mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -38,27 +38,6 @@ private sealed trait Event {
 }
 
 private object DTO {
-
-  object MailboxPath {
```

Comment:
I don't agree, that's some DTO

Reply:
I'm the one who proposed this change.

The idea was to only keep the DTO related to Event. keeping other DTO here hurts readability (too much to review) and mixes concerns (mixing DTO of evets and other stuff do not seem good to me).

Maybe we can rename `DTO` to `EventDTOs` to clearly express this concern?

If you are strictly against these extraction to separate files, maybe separating `DTOs` & `EventDTOs` would satisfy us both?

Reply:
As discussed with @mbaechler , we came to the conclusion having a separated file for DTOs that are not Event was a good idea (but not one file per DTO). Can you group them in one `DTOs.scala` file?